modular_skyrat/modules/pixel_shift/code/pixel_shift.dm

```diff
@@ -33,8 +33,8 @@
 /mob/living/unpixel_shift()
 	if(is_shifted)
 		is_shifted = FALSE
-		pixel_x = body_position_pixel_x_offset
-		pixel_y = body_position_pixel_y_offset
+		pixel_x = body_position_pixel_x_offset + base_pixel_x
+		pixel_y = body_position_pixel_y_offset + base_pixel_y
```

Comment:
Oooh, that's probably the answer to my earlier question.

Reply:
yep